proxy/src/control/accept_policy.rs

```diff
@@ -0,0 +1,291 @@
+// use std;
+// use std::collections::{HashSet, VecDeque};
+// use std::collections::hash_map::{Entry, HashMap};
+// use std::net::SocketAddr;
+// use std::fmt;
+// use std::mem;
+use std::marker::PhantomData;
+
+use futures::{Async, Future, Poll, Stream};
+use futures_watch::Store;
+use indexmap::IndexSet;
+use ipnet::{Ipv4Net, Ipv6Net};
+use prost::Message;
+// use tower::Service;
+use tower_h2::{HttpService, BoxBody, RecvBody};
+use tower_grpc as grpc;
+
+use conduit_proxy_controller_grpc::{accept_policy as pb};
+use conduit_proxy_controller_grpc::accept_policy::client::{AcceptPolicy as Client};
+use conduit_proxy_controller_grpc::common::ip_address;
+
+use accept_policy::{self, EndpointMatch, Net};
+
+/// A handle to start watching a destination for address changes.
+pub struct AcceptPolicy<S>
+where
+    S: HttpService<RequestBody = BoxBody, ResponseBody = RecvBody>
+{
+    inbound: StreamPublish<InboundStreamCall, S>,
+    outbound: StreamPublish<OutboundStreamCall, S>,
+}
+
+/// Continually publishes messages from a stream response into a Watch.
+///
+/// If the stream terminates,
```

Comment:
_what happens if the stream terminates_???

Reply:
;) we just keep issuing the request forever.